components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx

```diff
@@ -141,7 +141,7 @@ export const WalletSettingsMenu = (props: Props) => {
 
   const onClickRoute = (route: WalletRoutes | AccountPageTabs) => {
     if (route === WalletRoutes.AddHardwareAccountModalStart && isPanel) {
-      chrome.tabs.create({ url: `chrome://wallet${route}` }, () => {
+      chrome.tabs.create({ url: `chrome://wallet${encodeURIComponent(route)}` }, () => {
```

Comment:
This looks wrong to me - looking the code this is to support urls like these:

![image](https://github.com/user-attachments/assets/2dc44148-91fa-4542-b7e5-742d852f5743)

which will break if they're URI encoded

Reply:
ahh ok, these are all hardcoded paths. Thanks yeah this should be removed